docs/tutorials/geomesa-examples-authorizations.rst

```diff
@@ -130,8 +130,8 @@ interface:
     }
 
 When a GeoMesa ``DataStore`` is instantiated, it will scan for available
-service providers. Third-party implementations can be enabled by placing
```

Comment:
oops, can we fix the source for this? The deleted lines are the correct phrasing.